security/s2a/internal/crypter/aesgcm.go

```diff
@@ -26,17 +26,21 @@ import (
 
 // Supported key sizes in bytes.
```

Comment:
Should we call it `aes16GcmKeySize` and `aes32GcmKeySize`? I left it as 128/256 since I've never seen it referred to in bytes rather than bits. Using 16/32 might be a bit unconventional, WDYT?

Reply:
Good catch! I think these are the only places where we should keep 128/256, because AES-128-GCM and AES-256-GCM are recognized names.